spec/features/system/admin/masquerades_spec.rb

```diff
@@ -17,7 +17,7 @@
 
       scenario 'I can masquerade a user' do
         visit admin_users_path
-        find("button.user-masquerade-#{user_to_masquerade.id}").click
+        find("a.user-masquerade-#{user_to_masquerade.id}").click
```

Comment:
If possible, maybe just remove the element type from the class selector altogether? Then `.user-masquerade-whatever` will stay resilient for whatever element we may choose in the future.